docs/reference/setup/install/docker.asciidoc

```diff
@@ -154,25 +178,24 @@ services:
       memlock:
         soft: -1
         hard: -1
-    mem_limit: 1g
     volumes:
```

Comment:
I don't think we should suggest this anymore because I believe it constrains the disk cache.